libleptongui/src/gschem_bottom_widget.c

```diff
@@ -1218,22 +1178,32 @@ gschem_bottom_widget_set_status_text_color (GschemBottomWidget *widget, gboolean
   g_return_if_fail (widget != NULL);
 
 #ifdef ENABLE_GTK3
-  const GdkRGBA* color = NULL;
-#else
+  GtkCssProvider *provider = gtk_css_provider_new ();
+  GtkStyleContext *context = gtk_widget_get_style_context (widget->status_label);
+  if (active)
+  {
+    gtk_css_provider_load_from_data (provider,
+                                     "label { color: green; }", -1, NULL);
```

Comment:
Did you decide to postpone taking into account the status active color configuration option (schematic.status-bar::status-active-color)?

Reply:
This was my oversight, sorry.  Hopefully it is fixed in the last commit here.

Reply:
It works, thank you.